server/src/routes/schedule.js

```diff
@@ -57,15 +59,35 @@ router.get('/candidates', async (req, res) => {
   });
 });
 
+/**
+ * Get a candidate's firstname, for the candidate availabilty page.
+ *
+ * This should be the only method that can be used without authentication.
+ */
+router.get('/candidate/name/:uuid', (req, res) => {
+  const { uuid } = req.params;
+  const sql = 'SELECT firstName FROM Candidate WHERE uuid = ?';
+  const sqlcmd = connection.format(sql, [uuid]);
+  connection.query(sqlcmd, (err, result) => {
+    if (err) {
+      return res.status(400).send({ message: 'Internal server Error.' });
```

Comment:
I think the status code for internal server error should be 5xx.

Reply:
Yeah, typo on my part.

Reply:
Changed to 500